tests/perf/pubsub_publish_http/test.js

```diff
@@ -0,0 +1,107 @@
+/*
+Copyright 2022 The Dapr Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import http from "k6/http";
+import { check } from "k6";
+import crypto from "k6/crypto";
+
+const KB = 1000
+const MAX_MS_ALLOWED = 1
+// padd with leading 0 if <16
+function i2hex(i) {
+    return ('0' + i.toString(16)).slice(-2);
+}
+
+function randomStringOfSize(size) {
+    const bytes = crypto.randomBytes(size);
+    const view = new Uint8Array(bytes);
+    return view.reduce(function (memo, i) { return memo + i2hex(i) }, '')
```

Comment:
https://github.com/ItalyPaleAle/arraybuffer for next time :)

Reply:
It would be an option if k6 allows node packages, things gets a bit complicated when using external dependencies. Unless you want to convert your library into a k6 module, otherwise I think I'll wait for a next time yet